bundles/org.openhab.binding.iaqualink/src/main/java/org/openhab/binding/iaqualink/internal/api/model/AccountInfo.java

```diff
@@ -0,0 +1,236 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.iaqualink.internal.api.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Account Info Object
+ *
+ * @author Dan Cunningham - Initial contribution
+ *
+ */
+public class AccountInfo {
+
+    @SerializedName("id")
+    @Expose
+    private Integer id;
+    @SerializedName("email")
+    @Expose
+    private String email;
+    @SerializedName("created_at")
```

Comment:
Consider using https://futurestud.io/tutorials/gson-builder-basics-naming-policies#fieldnamingpolicylower_case_with_underscores it will make it way less verbose (and less error prone.)

Reply:
I'm not sure i understand what you are asking here

Reply:
I went ahead and removed them

Reply:
Ah, i should have read this more carefully, i did not realize `FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES` was an option when creating a GSON builder, nice, i will take a look